src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -55,8 +43,34 @@ public void Initialize(ExtensionConfigContext context)
             }
             var converterManager = context.Config.GetService<IConverterManager>();
             converterManager = converterManager.AddNotificationHubConverters();
-            var provider = new NotificationHubAttributeBindingProvider(context.Config.NameResolver, converterManager, this);
+            var provider = new NotificationHubAttributeBindingProvider(converterManager, this);
             context.Config.RegisterBindingExtension(provider);
         }
+
+        internal static string GetSettingFromConfigOrEnvironment(string key)
```

Comment:
GetSettingFromConfigOrEnvironment looks like a common helper. Can we this shared across extensions?